src/datafactory/azext_datafactory/generated/_params.py

```diff
@@ -0,0 +1,633 @@
+        c.argument('access_resource_path', help='The resource path to get access relative to factory. Currently only em'
+                   'pty string is supported which corresponds to the factory resource.')
+        c.argument('profile_name', help='The name of the profile. Currently only the default is supported. The default '
+                   'value is DefaultProfile.')
+        c.argument(
+            'start_time', help='Start time for the token. If not specified the current time will be used.')
+        c.argument('expire_time', help='Expiration time for the token. Maximum duration for the token is eight hours an'
+                   'd by default the token will expire in eight hours.')
+
+    with self.argument_context('datafactory factory get-git-hub-access-token') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', options_list=['--name', '-n'], help='The factory name.', id_part='name')
+        c.argument('git_hub_access_code', help='GitHub access code.')
+        c.argument('git_hub_client_id', help='GitHub application client ID.')
+        c.argument('git_hub_access_token_base_url',
+                   help='GitHub access token base URL.')
+
+    with self.argument_context('datafactory integration-runtime list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.')
+
+    with self.argument_context('datafactory integration-runtime show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.', id_part='name')
+        c.argument('integration_runtime_name', options_list=['--name', '-n'], help='The integration runtime name.',
+                   id_part='child_name_1')
+        c.argument('if_none_match', help='ETag of the integration runtime entity. Should only be specified for get. If '
+                   'the ETag matches the existing entity tag, or if * was provided, then no content will be returned.')
+
+    with self.argument_context('datafactory integration-runtime linked-integration-runtime create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('factory_name', help='The factory name.')
+        c.argument('integration_runtime_name',
+                   help='The integration runtime name.')
+        c.argument('name', help='The name of the linked integration runtime.')
+        c.argument('subscription_id',
+                   help='The ID of the subscription that the linked integration runtime belongs to.')
+        c.argument('data_factory_name', help='The name of the data factory that the linked integration runtime belongs '
+                   'to.')
+        c.argument('data_factory_location', help='The location of the data factory that the linked integration runtime '
+                   'belongs to.')

```

Comment:
Same here. Can we name the option to be `--location` and use `get_location_type`?

Reply:
Are all location related parameter should be named as location?

Reply:
This should have changed now. Please review again. Thx